token-metadata/program/ProgrammableNFTGuide.md

```diff
@@ -257,11 +257,46 @@ The new unified api of token metadata exposes a system of delegations where othe
 
 #### Note: For programmable NFTS, auth rules manages which actors can become any of these types of delegates.
 
-* Collection - Allows an actor to verify and un-verify items in a asset grouping(collection)
-* Uses - Allows an Actor to "use" the asset and decrement the uses counter on chain which is how applications can implement specific limited or tracking behaviors.
+
+
+
+
+There are two types of delegates on Token Metadata: `TokenDelegate` and `MetadataDelegate`. 
+
+`TokenDelegate`s are delegates that operate at the token level – i.e., they are spl-token delegates. This allows the delegate to perform operations on the token account (burn, transfer, freeze). There can only be one token delegate at a time and they do not have an individual delegate account associated – their information is stored on the `TokenRecord` account. The `TokenDelegate`s are as follows.
+
 * Transfer - The same thing as an spl token delegate. This delegate can transfer the token, while delegate the owner can also transfer. Token Metadata will manage the delegate and provide the correct functionality regardless of Token Standard. When the delegate has transfered the 'amount' delegated the delegate is cleared. 
```

Comment:
`while delegate the owner can also transfer.`
remove "delegate"
`while the owner can also transfer`

Reply:
or `and while delegated, the owner can also transfer`